### ISSUE TITLE
Play next episode automatically bugged since 10.9.0

### ISSUE THREAD

**Issue by kraM1tt** [jellyfin/jellyfin-android#1384]
### Describe the bug

Despite having the option unticked, it continues to play the next episodes in the series infinitely.

![Screenshot_20240514_133003_Jellyfin](https://github.com/jellyfin/jellyfin-android/assets/89113987/2e33e6b9-078e-4381-83bc-a57f51b10932)

It worked perfectly before the server update. I usually fall asleep, the episode finishes and my phone auto locks itself, but now I wake up to a dead phone, the JF payback reporting log confirms it plays episodes all night.

Thanks

### Application version

2.6.1

### Where did you install the app from?

Google Play

### Device information

Samsung S21

### Android version

Android 14, one ui 6.0

### Jellyfin server version

10.9.0

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [X] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by kraM1tt** [comment]
Not sure if it helps but I just tested Findroid and it obeys the setting. Also Jellyfin for Tizen OS obeys the setting, so it does seem to be an Jellyfin Android issue

**Comment by Crash1602** [comment]
I have the same problem with a Google Pixel 8 and Google Pixel 8 Pro. It plays one episode after the other without stopping. Jellyfin Server 10.9.3 on Windows 11

**Comment by ConnorS1110** [comment]
Does this still happen for anyone in this thread? I had fixed an issue that sounds ~very~ similar to this in [#5398](https://github.com/jellyfin/jellyfin-web/pull/5398) in the web repo

**Comment by Crash1602** [comment]
Hi,
I tested the problem a few days ago with Jellyfin Server 10.9.6 and the AndroidApp 2.6.1 on both Pixel Smartphones. The Problem still exists with the integrated Player, but there are no Problems with the WebPlayer and it works as expected.

**Comment by kraM1tt** [comment]
Yes still broken for me, have to use Findroid to get around it, Web player has issues direct playing a lot of things

**Comment by dahamsta** [comment]
Also experiencing this, v. annoying.

**Comment by Crash1602** [comment]
I hope there will be a fix soon :-) because the WebPlayer is not as convenient as the integrated Player.

**Comment by kraM1tt** [comment]
Bumping before it gets auto closed

**Comment by BotBlake** [comment]
After testing on my end this issue no longer persists.
It did take a full restart of the app to take effect, but now it no longer autoplays the next Episode.

Closing as fixed for now.
If anyone here still has this issue please comment below and we will re-open it 👍

**Comment by LinAGKar** [comment]
This is still happening for me, with Jellyfin 10.10.7, Jellyfin Android 2.6.2 and integrated player

**Comment by Crash1602** [comment]
> This is still happening for me, with Jellyfin 10.10.7, Jellyfin Android 2.6.2 and integrated player

I suspect the fix will only be included in[ 2.7.0](https://github.com/jellyfin/jellyfin-android/milestone/6).

**Comment by BotBlake** [comment]
@LinAGKar @Crash1602 
Please retry with the latest 2.6.3 release.

I have been testing with 2.6.3-beta 😅

**Comment by gregersn** [comment]
I have this issue with Jellyfin 10.10.7 and Jellyfin Android 2.6.3
No "warning" that the next episode will play, setting for automatically play next is sett to  disabled, but next episodes still play.

**Comment by Crash1602** [comment]
The same thing happened to me. I was, and still am, hoping that it would be fixed in 2.7.0, because it was included in the plan.

**Comment by nulian** [comment]
Encountered the same problem this is not fixed at least in 2.6.3 with integrated player.

**Comment by Crash1602** [comment]
Is the issue actually still being watched by someone from the Jellyfin team, even though it is closed?

**Comment by mihailim** [comment]
This seems to be fixed in 2.7.0, in PR #1479 , so I don't think there's much point in reporting it for 2.6.x... We'll unfortunately have to wait for 2.7.0 to be released.

**Comment by Crash1602** [comment]
I want to kindly ask if there is any outlook on when version 2.7.0 might be released.
I’m not trying to push, i’m simply interested to know whether, for instance, testing phases are the reason it hasn’t been released yet.

Thanks!
